.github/workflows/ci-kubernetes-itests.yml

```diff
@@ -0,0 +1,108 @@
+name: Quarkus Kubernetes CI
+
+env:
+  MAVEN_ARGS: -B -e
+
+on:
+  push:
+    branches:
+      - main
+  pull_request_target:
+    types: [ opened, edited, labeled, reopened, synchronize ]
```

Comment:
Do you really need `edited`?

Reply:
Don't we wanna re-run when someone adds removes commits? Or that is not considered `edited`?

Reply:
It's the `synchronize` event. `edited` is typically if you edit the title or the description.

Reply:
I'll remove it then